config/configuration.json

```diff
@@ -964,5 +969,20 @@
       "difficulty":"NORMAL",
       "defaultPriority":"MEDIUM"
     }
+  },
+  "OceanBleedingCheck": {
+    "ocean.valid": "natural->strait,channel,fjord,sound,bay|harbour->*&harbour->!no|estuary->*&estuary->!no|bay->*&bay->!no|place->sea|seamark:type->harbour,harbour_basin,sea_area|water->bay,cove,harbour|waterway->artificial,dock",
+    "ocean.invalid": "man_made->breakwater,pier|natural->beach,marsh,swamp|water->marsh|wetland->bog,fen,mangrove,marsh,saltern,saltmarsh,string_bog,swamp,wet_meadow|landuse->*",
+    "highway.minimum": "path",
+    "highways.exclude": [
+      "bus_guideway"
+    ],
+    "lineItems.offending": "railway->rail,narrow_gauge,preserved,subway,disused,monorail,tram,light_rail,funicular,construction,miniature",
```

Comment:
Do offending and "invalid" have the same definitions? If so, we should stick to one naming convention

Reply:
They're not quite the same--offending refers to tags that must be found on a lineItem for it to be considered bleeding into the ocean. But if it were called valid/invalid I think that would make it less clear if they should be considered or not

Invalid refers to tags that cannot be found on an ocean for that ocean to be considered for the check.